app/components/message_draft_component.html.erb

```diff
@@ -6,26 +6,24 @@
 
       <%= render MessageDraftAttachmentsListComponent.new(message: @message) %>
 
-      <% if @message.not_yet_submitted? %>
         <div class="flex justify-between w-full">
-          <div class="gap-2 p-6 border-t-0 border-r-0 border-b border-l-0 border-gray-200">
+          <% if @message.correctly_created? %>
```

Comment:
`submittable?`

Reply:
Kedysi to tak bolo a menila sa (prehodnocovala) tato cast turbom po kazdej zmene (zobrazovalo sa tlacidlo odoslat a mizlo). Potom sme to ale chceli vyhodit, nech tam je to tlacidlo 'Odoslat' stale, ak bol validne vytvoreny draft. Ak nebude draft splnat este vsetky podmienky na odoslanie (napr. vsetky dokumenty na podpis uz podpisane), tak odoslanie neprejde (flash error).

Reply:
Toto je podla mna len vec nazvu, lebo neviem uplne co si predstavit pod `correctly_created?` co to vlastne znamena? Ze to je validne a da sa to odoslat?